daisy_workflows/image_build/debian/debian_10_fai.wf.json

```diff
@@ -30,6 +33,33 @@
           "google_cloud_repo": "${google_cloud_repo}"
         }
       }
+    },
+    "copy-gcs-object": {
+      "CopyGCSObjects": [
+        {
+          "Source": "${OUTSPATH}/root.tar.gz",
+          "Destination": "${OUTSPATH}/root.tar.gz"
+        }
+      ]
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "debian-10-v${build_date}",
```

Comment:
Please use the name and descriptions we are using for the regular publication and add the relevant licenses and Guest OS features.

Reply:
I will refer publish.json `description, `licenses` and `guestOSfeatures` fields. For license, I will use prod licences. In description, use ${build_date} replace {{time}}.